#! /bin/sh /usr/share/dpatch/dpatch-run
## 65_fix_release_error_with_postfix.dpatch by Alexander Wirt <formorer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix provided by Mark Martinec

@DPATCH@
diff -urNad amavisd-new~/amavisd amavisd-new/amavisd
--- amavisd-new~/amavisd	2007-06-27 12:43:00.000000000 +0200
+++ amavisd-new/amavisd	2007-09-17 23:00:25.000000000 +0200
@@ -13284,6 +13284,8 @@
       { ($smtp_s,$smtp_es,$msg) = ('450', '4.5.0', "ERROR: $err") }
     elsif ($resp =~ /^([1-5]\d\d) ([245]\.\d{1,3}\.\d{1,3})(?: |\z)(.*)\z/s)
       { ($smtp_s,$smtp_es,$msg) = ($1,$2,$3) }
+    elsif ($resp =~ /^(([1-5])\d\d)(?: |\z)(.*)\z/s)
+      { ($smtp_s,$smtp_es,$msg) = ($1, "$2.0.0" ,$3) }
     else
       { ($smtp_s,$smtp_es,$msg) = ('450', '4.5.0', "Unexpected: $resp") }
     push(@response, proto_encode('setreply',$smtp_s,$smtp_es,$msg));
